docs/tools-release-process.md

```diff
@@ -53,6 +53,8 @@ This document describes how to release all tools packages at once but this is no
 
 (with `cordova-common` -> `cordova-registry-mapper` removed from `master` branch of `cordova-common`)
 
+NOTE: Whenever we start a new major release it is possible that the `master` branch of Cordova packages will have explicit dependency on previous patch release of other packages.
```

Comment:
Why is this a problem? If tests are passing, this is obviously fine and there is no real need to update.

If we want to keep those dependencies updated, this should be a separate step in the release process or a general guideline.